applications/coreutils/rmdir.cpp

```diff
@@ -0,0 +1,33 @@
+#include <libsystem/io/Filesystem.h>
+#include <libsystem/io/Directory.h>
+#include <libsystem/io/Stream.h>
+
+Result rmdir(const char *path)
+{
+    if(!directory_exist(path))
+    {
+        printf("rmdir: failed to remove '%s': No such file or directory", path);
+        return ERR_NO_SUCH_FILE_OR_DIRECTORY;
+    }
+
+    Result result = filesystem_rmdir(path);
+    
+    if (result == ERR_NOT_READABLE)
+    {
+        printf("rmdir: failed to remove '%s': Not readable", path);
```

Comment:
Very nice work :+1:

You should use the method `result_to_string` of the `Result` class instead of those if statements

Reply:
Oh okay I'll do that 👍